central/image/datastore/datastore_impl.go

```diff
@@ -183,9 +190,15 @@ func (ds *datastoreImpl) GetImage(ctx context.Context, sha string) (*storage.Ima
 // GetImagesBatch delegates to the underlying store.
 func (ds *datastoreImpl) GetImagesBatch(ctx context.Context, shas []string) ([]*storage.Image, error) {
 	var imgs []*storage.Image
-	if ok, err := imagesSAC.ReadAllowed(ctx); err != nil {
-		return nil, err
-	} else if ok {
+	var err error
+	ok := true
+	if !features.PostgresDatastore.Enabled() {
+		if ok, err = imagesSAC.ReadAllowed(ctx); err != nil {
```

Comment:
Not sure I see the need to remove these for now, these are almost free checks

Reply:
Still seems cleaner to remove them. More than the performance, it's about readability -- someone in the future will see this and be confused as to whether they are necessary or not

Reply:
@md2119 I think this might be better accomplished by making `imagesSAC` something that always returns `true, nil` if the postgres flag is enabled, rather than branching everywhere